kr/kr.go

```diff
@@ -17,7 +17,7 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
-
+	
```

Comment:
nit: extra whitespace. running `gofmt` on each changed file should clean up any formatting issues